src/ConfigUtil.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Drupal\oe_bootstrap_theme;
+
+use Drupal\Core\Config\FileStorage;
+use Drupal\Component\Utility\Crypt;
+
+/**
+ * Utility class to import configuration from file.
+ */
+class ConfigUtil {
+
+  /**
+   * Installs configuration from file.
+   *
+   * @param string $module
+   *   Contains the name of the module whose configuration will be installed.
+   * @param string $path
+   *   Contains the folder where the config resides.
+   * @param array $config_names
+   *   Contains the name of configuration files that must be installed.
+   */
+  public static function importConfigFromFile(string $module, string $path, array $config_names): void {
+    $storage = new FileStorage(drupal_get_path('module', $module) . $path);
```

Comment:
use the drupal list service instead of drupal_get_path (deprecated since drupal 9.3 https://www.drupal.org/node/2940438)